apps/studio/src/lib/db/clients/bigquery.js

```diff
@@ -10,9 +10,17 @@ import { BigQueryOptions } from '@/common/appdb/models/saved_connection';
 import { table, time } from 'console';
 import { data } from 'jquery';
 import { buildDeleteQueries, buildInsertQueries, buildUpdateQueries, buildInsertQuery, genericSelectTop, buildSelectTopQuery, escapeString, joinQueries, escapeLiteral, applyChangesSql } from './utils';
+import { wrapIdentifier } from './mysql';
```

Comment:
General question: Is Big Query like a weird cousin of mysql? I know next to nothing about this DB

Reply:
It is not, good catch lol. Auto import did me dirty

Reply:
This looks to be still an issue?